configure

```diff
@@ -224,9 +223,9 @@ while : ; do
     -with-flambda-invariants|--with-flambda-invariants)
         with_flambda_invariants=true;;
     -with-cplugins|--with-cplugins)
-        with_cplugins=true;;
+        err "--with-cplugins is no longer supported";;  # Transitional?
     -no-cplugins|--no-cplugins)
-        ;; # Ignored for backward compatibility
+        ;; # Ignored for backward compatibility; transitional?
```

Comment:
Alternatively, the `-with-cplugins` option could be removed altogether, causing a generic error.  I have no opinion.

Reply:
In general, I think it’s nicer to have the flag preserved with the warning. Apart from anything else, if anything like it is ever reintroduced, it means care is implied not to re-use the old flag name without considering whether the new flag has the same meaning as the old one (largely academic I expect here, of course)

Reply:
I also think it's better to keep the error (and even remove the "transitional" comments.